src/components/Login.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useContext } from "react";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import { createPatient } from "../firebase";
+import { jsPsych } from "jspsych-react";
+import "../App.css";
+import "bootstrap/dist/css/bootstrap.css";
+import "@fortawesome/fontawesome-free/css/all.css";
+import LoginContext from "../contexts/LoginContext";
```

Comment:
@broarr is a context the right way to handle the passing back up of state? or could an action be passed as a prop? (not sure what the current react-idiomatic way is)